composer.json

```diff
@@ -13,8 +13,8 @@
         "psr-4": { "Aligent\\BraintreeBundle\\": "./src/" }
     },
     "require": {
-        "braintree/braintree_php": "3.36.*",
-        "php": ">=7.1",
+        "braintree/braintree_php": "6.4.*",
```

Comment:
This is a pretty major version bump, have we checked the changelog for possible breaking changes?
Specifically version 4, 5, and 6:
https://github.com/braintree/braintree_php/blob/master/CHANGELOG.md
(It's unlikely that anything broke here given that you were able to place a full transaction, but it's worth checking just in case)

Reply:
See my more detailed upgrade comment below: https://github.com/aligent/braintree-orocommerce/pull/20#issuecomment-905129146